.github/workflows/dlp-pipelines.yml

```diff
@@ -349,12 +349,20 @@ jobs:
         run: |
           inspect_job_data=$(gcloud dataflow jobs list --project ${{env.PROJECT_ID}} --status active --format json --filter="name=${{env.INSPECT_JOB_NAME}}")
           inspect_job_id=$(echo "$inspect_job_data" | jq -r '.[].id')
-          gcloud dataflow jobs cancel $inspect_job_id --project ${{env.PROJECT_ID}}
+          if !$inspect_job_id; then
```

Comment:
@chitara-01 https://stackoverflow.com/questions/58858429/how-to-run-a-github-actions-step-even-if-the-previous-step-fails-while-still-f can we also add these in the cleanup jobs to make sure that clean up always succeeds even if one step fails? The point is to make sure cleanup job overall always succeeds even if one particular step fails. The suggestion is keeping in mind the overall scenario when more steps would be needed as part of cleanup for other formats/workflows.